help/user-guide/generating-the-isa-structure.md

```diff
@@ -81,4 +81,4 @@ A modelling assay is described and linked using the following fields:
 * [Creators](general-attributes.html#creators)
```

Comment:
```suggestion
* [Creators]({{ "general-attributes#creators" | relative_url }})
```

Reply:
Idem! These are relative links, which should not be piped in the relative_url method.